### ISSUE TITLE
aiohttp-2.x.x wheels for Python 3.5 include .so files linked against Python 3.4

### ISSUE THREAD

**Issue by shargan** [aio-libs/aiohttp#2057]
## Long story short

Trying to use aiohttp-2.2.3 on a host with only Python 3.5 fails because the wheel includes a C extension linked against Python 3.4 libraries. This is the case for all aiohttp-2.x.x-cp35-cp35m-manylinux1_x86_64 wheels at pypi.

## Expected behaviour

C extensions should only be linked against the version of Python the wheel is built against.

## Actual behaviour

The following wheels include aiohttp/_http_parser.cpython-34m.so which is linked to libpython3.4m.so.1.0:
- [2.2.0](https://pypi.python.org/packages/2b/b8/f8331846fc582bb8c24c0425cfa32bf5d00446be080316cf2a7296fbd6e0/aiohttp-2.2.0-cp35-cp35m-manylinux1_x86_64.whl#md5=c1b4d78f0874fc4a168e8ce8a2708121)
- [2.2.1](https://pypi.python.org/packages/46/14/1a6cf317e8981a50898e7a272181f83071375b9998ba242ad13f39b9455b/aiohttp-2.2.1-cp35-cp35m-manylinux1_i686.whl#md5=96d16b665583cf4e9af2d90d766765f6)
- [2.2.2](https://pypi.python.org/packages/18/9f/34806d8a15e20e7ee73d9ae53cef40760f575923c6531d4691b35af3ef80/aiohttp-2.2.2-cp35-cp35m-manylinux1_x86_64.whl#md5=128c6c1e677fa12a5d6affa0cfe9e09c)
- [2.2.3](https://pypi.python.org/packages/8b/f9/93f8393ff83575d1cbfef65ddde523d77c0e54ca7debd0c52bedf30e5a37/aiohttp-2.2.3-cp35-cp35m-manylinux1_x86_64.whl#md5=bc7f682c3ea75a01f31d34f35daabfa7)

## Steps to reproduce

```bash
$ mkdir tmp
$ cd tmp
~/tmp$ curl -s -o aiohttp-2.2.3.whl https://pypi.python.org/packages/8b/f9/93f8393ff83575d1cbfef65ddde523d77c0e54ca7debd0c52bedf30e5a37/aiohttp-2.2.3-cp35-cp35m-manylinux1_x86_64.whl#md5=bc7f682c3ea75a01f31d34f35daabfa7
~/tmp$ unzip -q aiohttp-2.2.3.whl

# this is the offender
~/tmp$ ldd aiohttp/_http_parser.cpython-34m.so
	linux-vdso.so.1 =>  (0x00007fff16593000)
	libpython3.4m.so.1.0 => not found
	libpthread.so.0 => /lib/x86_64-linux-gnu/libpthread.so.0 (0x00007f76e321a000)
	libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f76e2e55000)
	/lib64/ld-linux-x86-64.so.2 (0x00007f76e365b000)

# others are fine
~/tmp$ find aiohttp -name \*.so | xargs ldd
aiohttp/_http_parser.cpython-34m.so:
	linux-vdso.so.1 =>  (0x00007fff411d1000)
	libpython3.4m.so.1.0 => not found
	libpthread.so.0 => /lib/x86_64-linux-gnu/libpthread.so.0 (0x00007f2f5ecc5000)
	libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f2f5e900000)
	/lib64/ld-linux-x86-64.so.2 (0x00007f2f5f106000)
aiohttp/_frozenlist.cpython-35m-x86_64-linux-gnu.so:
	linux-vdso.so.1 =>  (0x00007ffdcabeb000)
	libpthread.so.0 => /lib/x86_64-linux-gnu/libpthread.so.0 (0x00007fc1cf1ea000)
	libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007fc1cee25000)
	/lib64/ld-linux-x86-64.so.2 (0x00007fc1cf614000)
aiohttp/_http_parser.cpython-35m-x86_64-linux-gnu.so:
	linux-vdso.so.1 =>  (0x00007ffd11bf7000)
	libpthread.so.0 => /lib/x86_64-linux-gnu/libpthread.so.0 (0x00007f7a0e182000)
	libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f7a0ddbd000)
	/lib64/ld-linux-x86-64.so.2 (0x00007f7a0e5c3000)
aiohttp/_websocket.cpython-35m-x86_64-linux-gnu.so:
	linux-vdso.so.1 =>  (0x00007fff8ff70000)
	libpthread.so.0 => /lib/x86_64-linux-gnu/libpthread.so.0 (0x00007f936c9b2000)
	libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f936c5ed000)
	/lib64/ld-linux-x86-64.so.2 (0x00007f936cdd5000)
```

## Your environment

aiohttp-2.2.3 on Ubuntu Xenial with Python 3.5.

**Comment by fafhrd91** [comment]
@asvetlov I think 2.0 was fine

**Comment by shargan** [comment]
@fafhrd91 2.0.0 has the same problem:
```$ mkdir tmp
$ cd tmp/
~/tmp$ curl -s -o aiohttp-2.0.0.whl https://pypi.python.org/packages/4b/6e/947276936e520a532e549b96332988fa0ebd20955eecd36f12fb9ce154f6/aiohttp-2.0.0-cp35-cp35m-manylinux1_x86_64.whl#md5=fe24423ad518aafe90696e1787ec5583
~/tmp$ unzip -qq aiohttp-2.0.0.whl

~/tmp$ find aiohttp -name \*.so
aiohttp/_websocket.cpython-35m-x86_64-linux-gnu.so
aiohttp/_http_parser.cpython-35m-x86_64-linux-gnu.so
aiohttp/_http_parser.cpython-34m.so

~/tmp$ ldd aiohttp/_http_parser.cpython-34m.so
	linux-vdso.so.1 =>  (0x00007fff2b5fe000)
	libpython3.4m.so.1.0 => not found
	libpthread.so.0 => /lib/x86_64-linux-gnu/libpthread.so.0 (0x00007fef54109000)
	libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007fef53d43000)
```

**Comment by asvetlov** [comment]
Fixed in 2.3 alpha
